openapi/openapi.yaml

```diff
@@ -934,6 +934,30 @@ paths:
             default: false
           description: >
             Do not use cached results, always run adviser.
+        - name: github_event_id
```

Comment:
@pacospace to be consistent, do we want to prefix these with `github`? also in thamos workflow.

Reply:
Yes I like it, but do we keep `revision` at the end?

Reply:
Just `revision`? Without `github_` prefix?